mediacapture-record/MediaRecorder-bitrate.https.html

```diff
@@ -5,6 +5,9 @@
 <link rel="help" href="https://w3c.github.io/mediacapture-record/MediaRecorder.html">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src=/resources/testdriver.js></script>
+<script src=/resources/testdriver-vendor.js></script>
+<script src='../mediacapture-streams/permission-helper.js'></script>
```

Comment:
can we please have one convention on how filenames are quoted in "script src=" statements?
This makes 3 different versions in a single file.